src/games/strategy/triplea/ai/proAI/data/ProOtherMoveOptions.java

```diff
@@ -52,6 +54,7 @@ public String toString() {
 
   private static Map<Territory, ProTerritory> createMaxMoveMap(final List<Map<Territory, ProTerritory>> moveMaps,
       final PlayerID player, final boolean isAttacker) {
+    final GameData data = ProData.getData();
```

Comment:
ProData.getData() is essentially a global AI variable. Its holding the game data for the current phase. I tend to just put the declaration at the top of the method when its being used as it acts as almost an input parameter.